### PR TITLE
adding in/2021-03/11/ & adding in/2021-03/12/

### PR DESCRIPTION
**Pull request by animemoeus**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [x] Have you run `run.sh` or `node deploy.js -b test -l [language] -q [quarter]`?
- [x] Have you checked for semicolons in the titles?

_NOTE: these things are not required to open a PR and can be done
afterwards / while the PR is open._

For example, `node deploy.js -b test -l en -q 2018-02`.

### CODE REVIEW

**Comment by VitalikL** [comment]
@animemoeus Thanks for the contribution! Can you please check the formatting for `01.md` and `07.md` and compare to how it is done in `English` for example.

Cheers

File: src/in/2021-03/11/01.md [hunk 1]
```diff
@@ -3,17 +3,18 @@ title: "Merindukan Lebih Banyak"
 date: 04/09/2021
 ---
 
-### Untuk Pelajaran Pekan Ini Bacalah: 1 Kor. 10: 1-11; Im. 4: 32-35; Yoh. 1: 29; Ibr. 4: 1-11; Mzm. 95: 8-11.
+### Untuk Pelajaran Pekan Ini Bacalah:
+1 Kor. 10: 1-11; Im. 4:32-35; Yoh. 1: 29; Ibr. 4: 1-11; Mzm. 95:8-11.
 
 > <p>Ayat Hafalan</p>
-> "Semuanya ini telah terjadi sebagai contoh bagi kita untuk memperingatkan kita, supaya jangan kita menginginkan hal-hal yang jahat seperti yang telah mereka perbuat" (1 Kor. 10: 6).
+> "Semuanya ini telah terjadi sebagai contoh bagi kita untuk memperingatkan kita, supaya jangan kita menginginkan hal-hal yang jahat seperti yang telah mereka perbuat" (l Kor. 10: 6).
```
Comment (animemoeus) [inline src/in/2021-03/11/01.md]: Typo from 1 to

File: src/in/2021-03/11/01.md [hunk 2]
```diff
@@ -3,17 +3,18 @@ title: "Merindukan Lebih Banyak"
...
 
-Museum Seni Queens di New York, Amerika Serikat, menyimpan model arsitektur kota terbesar di dunia, menggambarkan semua bangunan di New York. Pada skala 1:1, 200 (di mana 2,5 cm atau 1 inci sesuai dengan 33 meter atau 100 kaki) itu mencakup hampir 870 meter persegi (9.335 kaki persegi). Awalnya diselesaikan pada tahun 1964 oleh 100 pengrajin yang telah bekerja selama lebih dari tiga tahun untuk menyelesaikan proyek. Itu telah diperbarui ke tahun 1990-an dan tidak mencerminkan pemandangan kota tahun 2021. Ini adalah salinan asli yang sangat rumit dan terperinci.
+Museum Seni Queens di New York, Amerika Serikat, menyimpan model arsitektur kota terbesar di dunia, menggambarkan semua bangunan di New York. Pada Skala 1:1, 200 (di mana 2,5 cm atau 1 inci sesuai dengan 33 meter atau 100 kaki) itu mencakup hampir 870 meter persegi (9.335 kaki persegi). Awalnya diselesaikan pada tahun 1964 oleh 100 pengrajin yang telah bekerja selama lebih dari tiga tahun untuk menyelesaikan proyek. Itu telah diperbarui ke tahun 1990-an dan tidak mencerminkan pemandangan kota tahun 2021. Ini adalah salinan asli yang sangat rumit dan terperinci.
```
Comment (animemoeus) [inline src/in/2021-03/11/01.md]: Wrong capitalization

File: src/in/2021-03/11/02.md [hunk 3]
```diff
@@ -3,16 +3,16 @@ title: "Menjadi Pengikut Musa"
 date: 05/09/2021
 ---
 
-`Bacalah 1 Kor. 10: 1-11. Apa yang Paulus ingin sampaikan kepada para pembacanya di Korintus ketika dia menyebut "contoh"?`
+`Bacalah 1 Kor. 10: 1—11. Apa yang Paulus ingin sampaikan kepada para pembacanya di Korintus ketika dia menyebut "contoh"?`
 
-Istilah Yunani yang digunakan dalam 1 Kor. 10: 6 (dan juga diadaptasi dalam bentuk yang serupa dalam 1 Kor. 10: 11), yang diterjemahkan sebagai "contoh" dalam kebanyakan terjemahan bahasa Inggris, adalah _typos_. Dalam bahasa Inggris kata _type_ didasarkan pada kata benda Yunani ini. _Type_ (atau contoh) tidak pernah yang asli tetapi semacam simbol atau gambaran dari yang asli. Ini adalah model dari sesuatu yang lain.
+Istilah Yunani yang digunakan dalam 1 Kor. 10: 6 (dan juga diadaptasi dalam bentuk yang serupa dalam 1 Kor. 10: 11), yang diterjemahkan sebagai "contoh" dalam kebanyakan terjemahan bahasa Inggris, adalah _typos._ Dalam bahasa Inggris kata _type_ didasarkan pada kata benda Yunani ini. _Type_ (atau contoh) tidak pernah yang asli tetapi semacam simbol atau gambaran dari yang asli. Ini adalah model dari sesuatu yang Iain.
```
Comment (animemoeus) [inline src/in/2021-03/11/02.md]: typo from lain to Iain

File: src/in/2021-03/11/02.md [hunk 4]
```diff
@@ -3,16 +3,16 @@ title: "Menjadi Pengikut Musa"
...
+Paulus menganggap pemberhentian-pemberhentian penting dari perjalanan padang belantara ini sebagai suatu tipe, atau contoh, dari baptisan individu. Dalam jejak logika Paulus, rujukan ke "makanan rohani" harus merujuk pada manna _(bandingkan dengan Kel. 16: 31—35)._ Israel minum dari batu karang, yang diidentifikasi Paulus sebagai Kristus _(1 Kor. 10: 4)._ Pikirkan tentang Yesus, misalnya, sebagai "roti kehidupan" _(Yoh. 6: 48)_ dan sebagai "air hidup" _(Yoh. 4: 10),_ dan ini semua masuk akal. Jadi, apa yang kita lihat di sini adalah penggunaan Paulus akan sejarah Perjanjian Lama sebagai contoh pengungkapan kebenaran rohani yang dapat diterapkan pada setiap orang Kristen dewasa ini.
 
-`Pikirkan kembali tentang pengalaman orang Israel dalam Keluaran. Apa pelajaran rohani yang dapat kita pelajari dari "contoh" mereka, baik yang baik maupun yang buruk yang mereka tinggalkan bagi kita?`
+`Pikirkan kembali tentang pengalaman orang Israel dalam Keluaran. Apa pelajaran rohani yang dapat kita pelajari dari "contoh" mereka, baik Yang baik maupun yang buruk yang mereka tinggalkan bagi kita?`
```
Comment (animemoeus) [inline src/in/2021-03/11/02.md:18]: Wrong capitalization from yang to Yang

File: src/in/2021-03/11/03.md [hunk 5]
```diff
@@ -3,16 +3,16 @@ title: "Ritual dan Korban"
...
 
 Ritual adalah komunikator yang sangat baik dari nilai-nilai dan informasi penting, dan itu perlu dipahami dalam konteksnya. Itu biasanya membutuhkan waktu tertentu, lokasi tertentu, dan urutan tindakan yang telah ditentukan untuk menjadi efektif. Memang, ketika kita membaca perintah-perintah Alkitab dalam Perjanjian Lama tentang korban, itu menjadi jelas bahwa Allah memberikan perincian yang sangat spesifik tentang apa yang bisa dikorbankan, dan kapan, di mana, dan ritual serta prosedur apa yang harus diikuti.
 
-Inti dari banyak ritual, tentu saja, adalah darah, menumpahkan, dan memercikkan darah. Ini tidak menarik, juga tidak seharusnya, karena itu berhadapan dengan hal yang paling buruk di alam semesta, dan itu adalah dosa.
+Inti dari banyak ritual, tentu saja, adalah darah, menumpahkan, dan mernercikkan darah. Ini tidak menarik, juga tidak seharusnya, karena itu berhadapan dengan hal yang paling buruk di alam semesta, dan itu adalah dosa.
```
Comment (animemoeus) [inline src/in/2021-03/11/03.md]: Mistyping from memercikan to mernercikkan

File: src/in/2021-03/11/03.md [hunk 6]
```diff
@@ -3,16 +3,16 @@ title: "Ritual dan Korban"
...
 
-`Pikirkan tentang betapa buruknya dosa sehingga itu menuntut korban, pengorbanan diri, dari Salah Satu anggota Keallahan, Yesus, untuk mengadakan pendamaian bagi dosa. Apa yang hal ini seharusnya ajarkan kepada kita tentang mengapa kita harus bersandar hanya pada kasih karunia dan bukan perbuatan? Lagipula, apa yang bisa kita tambahkan kepada apa yang telah dilakukan Kristus untuk kita?`
+`Pikirkan tentang betapa buruknya dosa sehingga itu menuntut korban, pengorbanan diri, dari Salah Satu anggota Keallahan, Yesus, untuk mengadakan pendamaian bagi dosa. Apa yang hal ini seharusnya ajarkan kepada kita tentang mengapa kita harus bersandar hanya pada kasih karunia dan bukan perbuatan? Lagipula, apa yang bisa kita tambahkan kepada apa yang telah Kristus untuk kita?`
```
Comment (animemoeus) [inline src/in/2021-03/11/03.md:18]: Missing sentence "dilakukan"

File: src/in/2021-03/11/04.md [hunk 7]
```diff
@@ -5,16 +5,16 @@ date: 07/09/2021
...
 
-Ibrani 4: 3 menunjuk pada hubungan erat antara iman dan istirahat. Kita dapat masuk ke dalam perhentian-Nya hanya ketika kita percaya dan mempercayai Dia yang menjanjikan istirahat dan yang dapat memenuhi janji ini, dan tentu saja itu adalah Yesus Kristus.
+Ibrani 4: 3 menunjuk pada hubungan erat antara iman dan istirahat. Kita dapat masuk ke dalam perhentian-Nya hanya ketika kita percaya dan memercayai Dia yang menjanjikan istirahat dan yang dapat memenuhi janji ini, dan tentu saja itu adalah Yesus Kristus.
 
-`Bacalah kembali Ibr. 4: 3. Apa masalah utama yang dirujuk dari orang orang tersebut? Pelajaran apa yang dapat kita ambil dari hal ini untuk diri kita sendiri, kita yang telah memiliki "Injil diberitakan juga kepada kita sama seperti kepada mereka" (Ibr. 4: 2)?`
+`Bacalah kembali Ibr. 4: 3. Apa masalah utama yang dirujuk dari orangorang tersebut? Pelajaran apa yang dapat kita ambil dari hal ini untuk diri kita sendiri, kita yang telah memiliki "Injil ... diberitakan juga kepada kita sama seperti kepada mereka" (Ibr. 4: 2)?`
```
Comment (animemoeus) [inline src/in/2021-03/11/04.md]: Typo from orang-orang to orangorang

File: src/in/2021-03/11/05.md [hunk 8]
```diff
@@ -3,20 +3,20 @@ title: "Jangan Keraskan Hatimu"
 date: 08/09/2021
 ---
 
-`Bacalah Ibr. 4: 4-7 dan Mzm. 95: 8-11. Apa amaran yang diberikan di sana, baik dalam kitab Mazmur dan kitab Ibrani, dan apa yang hal itu seharusnya sampaikan kepada kita, hari ini?`
+`Bacalah Ibr. 4: 4—7 dan Mzm. 95: 8-11. Apa amaran yang diberikan sana, baik dalam kitab Mazmur dan kitab ibrani, dan apa yang hal itü se harusnya sampaikan kepada kita, hari ini?`
```
Comment (animemoeus) [inline src/in/2021-03/11/05.md]: Missing sentences, invalid capitalization, and wrong character

File: src/in/2021-03/11/05.md [hunk 9]
```diff
@@ -3,20 +3,20 @@ title: "Jangan Keraskan Hatimu"
...
 
-Ibrani 4: 4-7 mengutip baik kisah penciptaan dan Mzm. 95: 11 dalam konteks berbicara tentang ketidaksetiaan orang Israel dan, karenanya, kegagalan mereka untuk masuk ke dalam istirahat yang Allah inginkan bagi mereka.
+Ibrani 4: 4—7 mengutip baik kisah penciptaan dan Mzm. 95: 11 dalam konteks berbicara tentang ketidaksetiaan orang Israel dan, karenanya, kegagalan mereka untuk masuk ke dalam istirahat yang Allah inginkan bagi mereka.
 
-Sesungguhnya, Mzm. 95: 8-11 menghubungkan pengalaman padang belantara Israel dengan istirahat Allah dan termasuk sumpah Ilahi bahwa Israel yang tidak beriman tidak akan masuk ke dalam perhentian-Nya, yang awalnya dikaitkan dengan Tanah Perjanjian.
+Sesungguhnya, Mzm. 95: 8—11 menghubungkan pengalaman padang belantara Israel dengan istirahat Allah dan termasuk sumpah ilahi bahwa Israel yang tidak beriman tidak akan maşuk ke dalam perhentian-Nya, yang awalnya dikaitkan dengan Tanah Perjanjian.
```
Comment (animemoeus) [inline src/in/2021-03/11/05.md]: Invalid charachter ş

File: src/in/2021-03/11/05.md [hunk 10]
```diff
@@ -3,20 +3,20 @@ title: "Jangan Keraskan Hatimu"
...
 
-Tentu saja, Israel memang memasuki Tanah Perjanjian. Generasi yang baru melintasi perbatasan dan, dengan pertolongan Allah, mengambil benteng-benteng negeri itu dan menetap di sana.
+Tentü saja, Israel memang memasuki Tanah Perjanjian. Generasi yang baru melintasi perbatasan dan, dengan pertolongan Allah, mengambil benteng-benteng negeri itu dan menetap di sana.
```
Comment (animemoeus) [inline src/in/2021-03/11/05.md]: Invalid character ü

File: src/in/2021-03/11/05.md [hunk 11]
```diff
@@ -3,20 +3,20 @@ title: "Jangan Keraskan Hatimu"
...
 
-Namun, mereka tidak masuk ke dalam istirahat Allah, dengan gagasan bahwa banyak orang tidak mengalami kenyataan keselamatan dalam Yesus karena kurangnya iman mereka dimanifestasikan oleh ketidaktaatan yang mencolok. Bahkan, meskipun istirahat dikaitkan dengan negeri, itu termasuk lebih dari sekadar tempat mereka tinggal.
+Namun, mereka tidak maşuk ke dalam istirahat Allah, dengan gagasan bahwa banyak orang tidak mengalami kenyataan keselamatan dalam Yesus karena kurangnya iman mereka dimanifestasikan oleh ketidaktaatan yang mencolok. Bahkan, meskipun istirahat dikaitkan dengan negeri, itu termasuk lebih dari sekadar tempat mereka tinggal.
```
Comment (animemoeus) [inline src/in/2021-03/11/05.md]: Invalid character ş

File: src/in/2021-03/11/05.md [hunk 12]
```diff
@@ -3,20 +3,20 @@ title: "Jangan Keraskan Hatimu"
...
 
-`Ibrani 4: 6 menyatakan bahwa mereka yang telah mendengar janji Ilahi tentang istirahat sejati tidak masuk karena ketidaktaatan. Apa hubungan antara ketidaktaatan dan tidak memasuki perhentian Allah?`
+`Ibrani 4: 6 menyatakan bahwa mereka yang telah mendengar janji ilahi tentang istirahat sejati tidak maşuk karena ketidaktaatan. Apa hubungan antara ketidaktaatan dan tidak memasuki perhentian Allah?`
```
Comment (animemoeus) [inline src/in/2021-03/11/05.md]: Wrong capitalization and invalid character ş

File: src/in/2021-03/11/05.md [hunk 13]
```diff
@@ -3,20 +3,20 @@ title: "Jangan Keraskan Hatimu"
...
 
-"Hari ini" mengekspresikan urgensi. "Hari ini" berarti bahwa tidak ada lagi berlengah-lengah. "Hari ini" menuntut sambutan dan keputusan sekarang.
+"Hari ini” mengekspresikan urgensi. "Hari ini” berarti bahwa tidak ada lagi berlengah-lengah. "Hari ini” menuntut sambutan dan keputusan sekarang.
 
-Paulus berpegang pada kata sēmeron, "hari ini," dan benar-benar menekankan betapa pentingnya kata itu dalam konteks perhentian. Mazmur 95: 7, 8, sementara itu, adalah satu amaran dan permohonan kepada umat Allah untuk tidak mengulangi kesalahan nenek moyang mereka dan gagal untuk masuk ke dalam istirahat sejati yang hanya ditemukan dalam keselamatan yang Allah tawarkan kepada kita.
+Paulus berpegang pada kata _semeron,_ "hari ini,” dan benar-benar menekankan betapa pentingnya kata itu dalam konteks perhentian. Mazmur 95: 7, 8, sementara itu, adalah satu amaran dan permohonan kepada umat Allah untuk tidak mengulangi kesalahan nenek moyang mereka dan gagal untuk masuk ke dalam istirahat sejati yang hanya ditemukan dalam keselamatan yang Allah tawarkan kepada kita.
```
Comment (animemoeus) [inline src/in/2021-03/11/05.md:20]: Typo from sēmeron to semeron

File: src/in/2021-03/11/06.md [hunk 14]
```diff
@@ -3,9 +3,9 @@ title: "Menaklukkan Kota Surgawi"
 date: 09/09/2021
 ---
 
-Perkembangan logis dari ide-ide kunci dalam Ibr. 4 menjadi sangat jelas ketika membaca Ibr. 4: 8-11. Yosua tidak memberi Israel istirahat. Konsekuensinya, karena Allah bukan pembohong, pasti ada "istirahat" lain yang tersisa bagi umat Allah. Kelompok ini bukan hanya terdiri dari orang-orang percaya Yahudi. Itu mencakup semua orang yang telah menerima Yesus sebagai Juruselamat pribadi mereka.
+Perkembangan logis dan ide-ide kunci dalam Ibr. 4 menjadi sangat jelas ketika membaca Ibr. 4: 8—11. Yosua tidak memberi Israel istirahat. Konsekuensinya, karena Allah bukan pembohong, pasti ada "istirahat" lain yang tersisa bagi umat Allah. Kelompok ini bukan hanya terdiri dari orang-orang percaya Yahudi. Itu mencakup semua orang yang telah menerima Yesus sebagai Juru Selamat pribadi mereka.
```
Comment (animemoeus) [inline src/in/2021-03/11/06.md]: Mistyping from from to and

Mistyping from Juruselamat to Juru Selamat

File: src/in/2021-03/11/06.md [hunk 15]
```diff
@@ -3,9 +3,9 @@ title: "Menaklukkan Kota Surgawi"
...
 
-`Bacalah Gal. 3: 26-29 dan catat karakteristik dari umat perjanjian Allah pasca salib. Apa artinya bahwa tidak ada orang Yahudi atau orang Yunani, tidak ada hamba atau orang merdeka, tidak ada laki-laki atau perempuan dalam konteks yang Paulus tuliskan?`
+`Bacalah Gal. 3: 26-29 dan catat karakteristik dari umat perjanjian Allah pascasalib. Apa artinya bahwa tidak ada orang Yahudi atau orang Yunani, tidak ada hamba atau orang merdeka, tidak ada laki-laki atau perempuan dalam konteks yang Paulus tuliskan?`
```
Comment (animemoeus) [inline src/in/2021-03/11/06.md]: Mistyping from `pasca `salib to and `pascasalib`

File: src/in/2021-03/11/07.md [hunk 16]
```diff
@@ -3,16 +3,16 @@ title: "Pendalaman"
...
 
-### Pertanyaan-Pertanyaan untuk Diskusi:
+**Pertanyaan-Pertanyaan untuk Diskusi:**
 
 `1. Apa yang istimewa dari Sabat hari ketujuh sehingga itu merupakan gambaran awal istirahat surgawi Allah bagi umat-Nya? Yaitu, bagaimana perhentian Sabat memberi kita rasa pendahuluan akan kekekalan?`
 
 `2. Pendamaian berarti rekonsiliasi dan menunjukkan jalan kembali kepada Allah. Pikirkan tentang pernyataan penting ini yang ditemukan dalam Rm. 5: 11: "Dan bukan hanya itu saja! Kita malah bermegah dalam Allah oleh Yesus Kristus, Tuhan kita, sebab oleh Dia kita telah menerima pendamaian itu." Jika seseorang bertanya kepada Anda, "Apa artinya diperdamaikan dengan Allah, dan apa perbedaan yang rekonsiliasi ini buat dalam hidup Anda," apa yang akan Anda jawab?`
 
-`3. Bagaimana kita dapat menghindari menjadi sibuk dalam hal-hal kecil dalam kehidupan Kristen kita? Apa yang membuat kita tetap fokus pada gambaran besar yang ditawarkan dalam Firman Allah?`
+`3. Bagaimana kita dapat menghindari menjadi sibuk dalam hal-hal kecil dalam kehidupan Kristen kita? Apa yang membuat kita tetap fokus pada gambaran beşar yang ditawarkan dalam Firman Allah?`
```
Comment (animemoeus) [inline src/in/2021-03/11/07.md:16]: Invalid character `ş`

File: src/in/2021-03/11/03.md [hunk 17]
```diff
@@ -3,16 +3,16 @@ title: "Ritual dan Korban"
 date: 06/09/2021
 ---
 
-Sistem ritual dan korban Perjanjian Lama, seperti yang ditemukan dalam kitab _Imamat_, memberikan lebih banyak contoh dari apa yang kita lihat kemarin—simbol-simbol Perjanjian Lama menunjuk pada kebenaran Perjanjian Baru. Meskipun pembaca modern dari Alkitab sering melewati ritual-ritual ini, ritual-ritual ini memang berisi banyak kebenaran-kebenaran rohani yang penting yang dapat sangat bermanfaat bagi mereka yang mempelajarinya.
+Sistem ritual dan korban Perjanjian Lama, seperti yang ditemukan dalam kitab _Imamat,_ memberikan lebih banyak contoh dari apa yang kita lihat kes marin—simbol-simbol Perjanjian Lama menunjuk pada kebenaran Perjanjian Baru. Meskipun pembaca modern dari Alkitab sering melewati ritual-ritual ini, ritual-ritual ini memang berisi banyak kebenaran-kebenaran rohani yang penting yang dapat sangat bermanfaat bagi mereka yang mempelajarinya.
```
Comment (animemoeus) [inline src/in/2021-03/11/03.md]: Mistyping from ` kemarin—simbol-simbol` to `kes marin—simbol-simbol `

File: src/in/2021-03/11/04.md [hunk 18]
```diff
@@ -5,16 +5,16 @@ date: 07/09/2021
...
+Bagian-bagian ini mengingatkan pembaca untuk mengambil pelajaran yang dipelajari dari tuntunan Allah di masa lalu dengan serius, "supaya jangan seorang pun jatuh karena mengikuti contoh ketidaktaatan itu juga" _(Ibr. 4: 11)._ Perhatikan, ini adalah kesempatan! Israel memang mendengar Injil, ayat melanjutkan, tetapi firman itu tidak menguntungkan mereka. Bukannya iman mereka dikuatkan dengan percaya dan penurutan, mereka memilih pendurhakaan _(bandingkan dengan Ibr. 3: 7—15),_ dan dengan demikian, mereka tidak pernah mengalami istirahat yang Allah inginkan untuk mereka.
 
-Ibrani 4: 3 menunjuk pada hubungan erat antara iman dan istirahat. Kita dapat masuk ke dalam perhentian-Nya hanya ketika kita percaya dan mempercayai Dia yang menjanjikan istirahat dan yang dapat memenuhi janji ini, dan tentu saja itu adalah Yesus Kristus.
+Ibrani 4: 3 menunjuk pada hubungan erat antara iman dan istirahat. Kita dapat masuk ke dalam perhentian-Nya hanya ketika kita percaya dan memercayai Dia yang menjanjikan istirahat dan yang dapat memenuhi janji ini, dan tentu saja itu adalah Yesus Kristus.
```
Comment (animemoeus) [inline src/in/2021-03/11/04.md]: Mistyping from `mempercayai` to `memercayai`

**Comment by VitalikL** [comment]
@animemoeus I meant the formatting of these:

`### Untuk Pelajaran Pekan Ini Bacalah` -> The list of Bible verses must be on the next line
`### Pertanyaan-Pertanyaan untuk Diskusi` -> Should be `**Pertanyaan-Pertanyaan untuk Diskusi**`

Also the questions in Friday, it is better to separate them with a new line.

**Review by animemoeus — COMMENTED** [review]
Hello

Sorry, I didn't mean to offend, and so on

But I worked meticulously and it took quite a while to make sure there were no spelling mistakes in the Sabbath school lessons.

I read it over and over again, wrote it manually, not using the help of text recognition because it could lead to some errors.

**Comment by animemoeus** [comment]
> @animemoeus I meant the formatting of these:
> 
> `### Untuk Pelajaran Pekan Ini Bacalah` -> The list of Bible verses must be on the next line
> `### Pertanyaan-Pertanyaan untuk Diskusi` -> Should be `**Pertanyaan-Pertanyaan untuk Diskusi**`
> 
> Also the questions in Friday, it is better to separate them with a new line.

Noted 👍

**Comment by VitalikL** [comment]
@animemoeus You are doing great work!

**Pull request merged**